Switched hh5 instances with xp65 (#941)
ACCESS-NRI/access-hive.org.au: docs/models/run_a_model/run_access-cm2.md

Files formatted as `<suite-name>a.xhist-<year><month><day>` contain metadata information.

## Troubleshooting
### Update suites still relying on hh5
Previously, the `hh5` NCI project hosted the Python environments and software often used within suites (e.g., `conda/analysis3` environemnts, `pythonlib/um2netcdf4` utility).<br>
Due to `hh5` decommissioning, all references to `hh5` must be replaced with the project storing the updated versions of the environments and software. For suites derived from the example suite above (`{{ suite_id }}` ), the updated versions are stored in `xp65`.

> 

Reviewer: ```suggestion

If you're not sure whether your suite relies on `hh5`, you can run the following command to check for any `hh5` occurrences from within the suite directory: `grep -r hh5 --exclude-dir=.svn.` 

```

> If your {{ model }} suite relies on `hh5`, update it by following the steps below:

Reviewer: ```suggestion
If your {{ model }} suite does have instances of `hh5`, update it by following the steps below:
```

#### 1. Update storage flags {: .no-toc }

> In the `suite.rc` file, look for all `-l storage` instances and replace `hh5` occurrencies with `xp65`

Reviewer: Typo

```suggestion
In the `suite.rc` file, look for all `-l storage` instances and replace `hh5` occurrences with `xp65`
```

#### 2. Update ocean_ke_check task {: .no-toc }
In the `suite.rc` file, within the `[[ocean_ke_check]]` task, update the module-loading lines to use `xp65` instead of `hh5`:
```diff
- module use /g/data/hh5/public/modules
+ module use /g/data/xp65/public/modules
module unload python
- module load conda/analysis3
+ module load conda/analysis3-25.05
```

!!! warning

>     The `conda/analysis3-25.05` version is the latest working version for {{ model }} setup. Therefore, we strongly recommend loading this version. For further information, refer to the [related Hive Forum post](https://forum.access-hive.org.au/t/issues-when-transitioning-from-hh5-to-xp65-in-suite-runs/4544).

Author: I followed the instructions from the related Hive Post that mentioned an issue with ACCESS-CM2 suites and versions greater than `conda/analysis3-25.05`. 

However, the default example suite `u-cy339` [loads `conda/analysis3`](https://code.metoffice.gov.uk/trac/roses-u/browser/c/y/3/3/9/trunk/suite.rc#L493) without any version, and I think it works fine.
Therefore, the issue might have been fixed in later `conda/analysis3` versions. 

I think it's safe to change this to `conda/analysis3` and change the warning (saying that, for reproducibility, it might be best to select a specific version)

I'll change it

## Troubleshooting
### Update suites still relying on hh5
!!! warning
    Some suites might still not work when ported this way.<br>
    If you have a suite that was relying on `hh5` and, even after following the steps below, the run submission fails, consider [getting help on the Hive Forum](/about/user_support/ask_on_forum).

Previously, the `hh5` NCI project hosted the Python environments and software often used within suites (e.g., `conda/analysis3` environemnts, `pythonlib/um2netcdf4` utility).<br>
Due to `hh5` decommissioning, all references to `hh5` must be replaced with the project storing the updated versions of the environments and software. For suites derived from the example suite above (`{{ suite_id }}` ), the updated versions are stored in `xp65`.

To check if your {{ model }} suite relies on `hh5`, run the following command from your suite directory:
```
grep -r hh5 --exclude-dir=.svn .
```

> Then, follow the steps below to ensure your suite is not using `hh5`.

Reviewer: I suggest merging this sentence with the one you had below under the first step. I think this is clearer, unless I've misunderstood something.

```suggestion
If you see any output from the command above, follow the steps below to replace all `hh5` occurrences with `xp65`.
```

#### 1. Replace hh5 with xp65 {: .no-toc }

> If you see any output from the command above, replace any `hh5` occurrency with `xp65`.<br>

Author: Related comment above
